libraries/AP_BoardConfig/canbus_driver.cpp

```diff
@@ -34,7 +34,7 @@ const AP_Param::GroupInfo AP_BoardConfig_CAN::Driver::var_info[] = {
     // @DisplayName: Enable use of specific protocol over virtual driver
     // @Description: Enabling this option starts selected protocol that will use this virtual driver
     // @Values{Copter,Plane,Sub}: 0:Disabled,1:UAVCAN,2:KDECAN,3:ToshibaCAN,4:PiccoloCAN
-    // @Values: 0:Disabled,1:UAVCAN,3:ToshibaCAN,4:PiccoloCAN
+    // @Values: 0:Disabled,1:UAVCAN,2:KDECAN,3:ToshibaCAN,4:PiccoloCAN
```

Comment:
Could use `@Values{Plane, copter, Sub}: ` as another line instead of making the change here.